mono/mini/interp/interp.c

```diff
@@ -5369,6 +5369,7 @@ interp_frame_iter_next (MonoInterpStackIter *iter, StackFrameInfo *frame)
 	/* This is the offset in the interpreter IR */
 	frame->native_offset = (guint8*)iframe->ip - (guint8*)iframe->imethod->code;
 	frame->ji = iframe->imethod->jinfo;
+	frame->managed = TRUE;
```

Comment:
@vargaz Do you think this is too disruptive ? Alternatively, I can explicitly check for FRAME_TYPE_INTERP in first_managed (where I had to consider them)

Reply:
I think its ok, frame->managed is not very well defined.